charts/osm/templates/validatingwebhook.yaml

```diff
@@ -18,6 +18,9 @@ webhooks:
   namespaceSelector:
     matchLabels:
       name: {{ include "osm.namespace" . }}
+  objectSelector:
+    matchLabels:
+      app.kubernetes.io/name: openservicemesh.io
```

Comment:
Shouldn't this include all the labels added by `{{- include "osm.labels" . | nindent 4 }}` in `osm-configmap.yaml`?

Reply:
Adding the other labels won't have any effect I don't think, since all the ConfigMaps in the control plane namespace will have those same label keys and values and only one control plane can exist in a namespace.

Reply:
+1 Especially since this is a cluster level resource, we should be as specific as possible with the selectors. Adding all the osm labels to the `matchLabels` list is a good idea
https://github.com/openservicemesh/osm/blob/main/charts/osm/templates/_helpers.tpl#L14-L16